Makefile

```diff
@@ -42,6 +42,27 @@ endif
 build:
 	$(GO) build $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS) -o $(PKGNAME) $(BUILDFILES)
 
+PLATFORMS := darwin linux windows
+
+WIN_FLAGS = -ldflags "-X main.Version=$(VERSION) -linkmode=internal -s -w -extldflags '-static' -extld=x86_64-w64-mingw32-gcc"
+
+buildNatives:
+	@for platform in $(PLATFORMS); \
+	do \
+		arch="amd64"; \
+		output_name="$(PKGNAME)-$$platform"; \
+		if [ $$platform = windows ]; then \
+			echo $$output_name; \
+			echo "asdasddassad"; \
+			wintags='$(TAGS) nolzma'; \
+			env CGO_ENABLED=1 CC=x86_64-w64-mingw32-gcc CXX=x86_64-w64-mingw32-g++ GOOS=$$platform GOARCH=$$arch $(GO) build $(WIN_FLAGS) -tags $$wintags $(BUILDV) $(BUILDFILES) ; \
+		else \
```

Comment:
Keywords can have newlines

Reply:
It's the makefile that needs the backslash, not the shell.